boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -173,6 +173,26 @@ application.
 Refer to :ref:`build_an_application` and :ref:`application_run` for
 more details.
 
+Downloading binary image
+========================
+
+A Linux host connected to the SPI interface of the ``intel_s1000_crb`` board
+can download a zephyr binary to RAM and execute the image.
+
+.. code-block:: console
+
+   cd <app-dir>/build
+   sudo -E python3 \
+   $ZEPHYR_BASE/boards/xtensa/intel_s1000_crb/support/download.py \
+   zephyr/zephyr.bin
+
+The script depends on a few python modules. These dependencies can be installed
+on the Linux host using the command below.
+
+.. code-block:: console
+
+   pip3 install --user pyyaml python-periphery hashlib bitstruct
```

Comment:
After the code block add:
```
Verify that ``~/.local/bin`` is at the beginning of your PATH (and add it
if it's missing) so these python modules can be found.
```

Reply:
Aren't these generic instructions for installation of python modules? It doesnt make sense to duplicate these in every doc page. If there is page that already has these instructions, I would rather point to it.

Reply:
Problem is that folks aren't reading the documentation when a link is provided. As a usability improvement, it's good to put critical information near where it's needed.  (I'm hunting down other stray uses of ``pip3 install --user`` and updating those too.)

Reply:
On second thought, you can leave this as is.  I'll come back around if needed to hunt down these uses of --user.